packages/web-api/src/methods.ts

```diff
@@ -1460,6 +1462,7 @@ export interface ChatUpdateArguments extends WebAPICallOptions, TokenOverridable
   attachments?: MessageAttachment[];
   blocks?: (KnownBlock | Block)[];
   link_names?: boolean;
+  metadata?: MessageMetadata;
   parse?: 'full' | 'none';
   file_ids?: string[];
   reply_broadcast?: boolean;
```

Comment:
Can you add `include_all_metadata` parameter to conversations.history API too? https://api.slack.com/methods/conversations.history#arg_include_all_metadata

Reply:
Good catch, this is done 👍